erc20_moc/Bet.noblock.js

```diff
@@ -42,6 +44,36 @@ class Bet {
     this.AMM_INTERACTION_TYPE = DIRECTION;
   }
 
+  /**
+   * Convert a quote to a number between 0 and 1
+   *
+   * @param value {number || string}
+   */
+  toUnitInterval(value) {
+    const input = new BigDecimal(value);
+    const one = new BigDecimal(1);
+    const oneToken = new BigDecimal(this.ONE);
+    const priceString = one.divide(input).multiply(oneToken).round(2);
+    const price = Number(priceString.value);
```

Comment:
do not use Number. floats are banned in this library :) use BigInt everywhere and decimals for prices. to store values in database use DECIMAL type

Reply:
I removed, it is already decimal in db. thanks